src/Client/BuzzHouse/Generator/StatementGenerator.cpp

```diff
@@ -2669,8 +2727,9 @@ void StatementGenerator::generateNextQuery(RandomGenerator & rg, SQLQueryInner *
         * static_cast<uint32_t>(collectionCount<SQLTable>(
                                     [](const SQLTable & t)
                                     {
+                                        /// I don't know what to do when the tables to exchange are on different clusters
```

Comment:
I feel the comment is missing the "so let's do X for now", whatever X is

Reply:
Fixed